public/index.html

```diff
@@ -10,9 +10,14 @@
     <div class="container">
         <h1>My To-Do List</h1>
         <div class="todo-app">
-            <input type="text" id="new-task" placeholder="Add a new task..." />
-            <button onclick="addTask()">Add Task</button>
-            <ul id="task-list"></ul>
+            <div class="add-task">
+                <input type="text" id="new-task" placeholder="Add a new task..." />
+                <button onclick="addTask()">Add Task</button>
+            </div>
+            <div class="task-list-section">
+                <input type="text" id="search-task" placeholder="Search tasks..." oninput="searchTask()" />
+                <ul id="task-list"></ul>
+            </div>
         </div>
     </div>
```

Comment:
# Use the node backend
@Msawekho from what I can see you are not utilizing the backend node server and doing everything on the fronend